po/vi.po

```diff
@@ -62,7 +62,7 @@ msgstr ""
 
 #: src/SUMMARY.md src/hello-world/what-is-rust.md
 msgid "What is Rust?"
-msgstr ""
+msgstr "Rust là gì?"
```

Comment:
Hi @ndminhvn and @qu-ngx, just a comment in case you haven't seen it: GitHub supports suggestions, which are great for fixing typos and suggesting other words. They look like this:

```suggestion
msgstr "Rust gì!"
```

and the reviewer or original author can quickly apply them on the PR page.

(You should of course just resolve this comment since it's just here to illustrate the concept :smile:)

Reply:
@mgeisler ah yes I know about GitHub's suggestion. Unfortunately, I don't recommend it since there might be some mistakes in translation and it did in this case. Better do it manually until this type of suggestion gets more complete and robust.

Reply:
Hi @mgeisler , thanks for your suggestion. We can definitely try to find the best translations based on both approaches (manual and GitHub's suggestions) 😄